### DISCUSSION TITLE
Redirect back with errors

### DISCUSSION THREAD

**Discussion by miquelangeld** [Laravel-Backpack/community-forum#1120] [Q&A (Help)] [answered]
Hi! I'm using a button in the list view which call a function, is it possible to do a redirect back with errors and have backpack show the error when the function return an error?

**Comment by pxpm** [comment, marked as answer]
Hey @miquelangeld 

To display alert messages (the ones on the top right corner), you can flash an alert to the session before the redirect: 
```php
Alert::error('you cant do this')->flash();
return redirect() .... 
```

Let me know if that helps 👍

**Reply by miquelangeld** [reply]
Works perfectly! Thanks  @pxpm
